content/search_api_3/latest/searching/offers/booking-availability.md

```diff
@@ -0,0 +1,47 @@
+---
+---
+
+# Booking availability
+
+The bookingAvailability indicates whether there are still tickets or seats left for an event or not. The following values for ‘bookingAvailability’ are applicable:
+
+| Value | Meaning |
+|--|--|
+| `Available` | there are still tickets and / or seats available for the event | 
+| `Unavailable` | the event is sold out or fully booked | 
+
+You can filter by address-related fields using two methods:
+* URL parameter
+* Advanced queries
+
+## URL parameter
+
+Using the `bookingAvailability` URL parameter, you can limit the results you get back to events with a certain bookingAvailability.
+Example to get results that have the bookingAvailability `Available`:
+
+```
+GET https://search.uitdatabank.be/offers/?bookingAvailability=Available
+```
+
+If used on its own, it will filter on the top-level status of the events/places.
+
+If used in combination with `dateFrom` and/or `dateTo` (see [date & time](https://documentatie.uitdatabank.be/content/search_api_3/latest/searching/offers/date.html)) it will combine the given status(es) with the given dateFrom and/or dateTo to query on subEvents, so you'll only get results that actually have the given bookingAvailability during the given date range.
```

Comment:
```suggestion
If used in combination with `dateFrom` and/or `dateTo` (see [date & time](https://documentatie.uitdatabank.be/content/search_api_3/latest/searching/offers/date.html)) it will combine the given availability with the given dateFrom and/or dateTo to query on subEvents, so you'll only get results that actually have the given bookingAvailability during the given date range.
```